lib/Clam/CfgBuilder.cc

```diff
@@ -1961,6 +2000,36 @@ void CrabIntraBlockBuilder::doMemIntrinsic(MemIntrinsic &I) {
   m_bb.havoc(m_lfac.mkIntVar(32), valueToStr(I));
 }
 
```

Comment:
It would be nice to add the comment.
Given an instruction `lhs := gep(ptr, offset)`  then the following CrabIR is added:
```
translate(lhs:= gep(ptr, offset));
lhs_offset := ptr_offset + offset;
lhs_size := ptr_size;
```